api/turing/api/pod_log_api.go

```diff
@@ -189,9 +188,18 @@ func getComponentType(defaultComponentType string, vars RequestVars, allowedType
 	return "", fmt.Errorf("Invalid component type '%s'", componentType)
 }
 
-type ListEnsemblingPodLogsOptions struct {
-	ProjectID *models.ID `schema:"project_id" validate:"required"`
+type podLogOptions struct {
 	ID        *models.ID `schema:"job_id" validate:"required"`
```

Comment:
Shall this be moved to `listEnsemblingPodLogsOptions` instead? I can imagine that it needs to be `schema:"router_id"` for router's pod logs options

Reply:
NVM, it was already changed in the next commit